modules/migration-mtc-release-notes-1-7.adoc

```diff
@@ -25,3 +25,4 @@ This release has the following known issues:
 * `MigPlan` custom resource does not display a warning when an AWS gp2 PVC has no available space. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1963927[*BZ#1963927*])
 * Direct and indirect data transfers do not work if the destination storage is a PV that is dynamically provisioned by the AWS Elastic File System (EFS). This is due to limitations of the AWS EFS Container Storage Interface (CSI) driver. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2085097[*BZ#2085097*])
 * Block storage for IBM Cloud must be in the same availability zone. See the link:https://cloud.ibm.com/docs/vpc?topic=vpc-block-storage-vpc-faq[IBM FAQ for block storage for virtual private cloud].
+* MTC 1.7.6 cannot migrate cron jobs from source clusters that support v1beta1 CronJob to OCP 4.12+ clusters, which do not support v1beta1 CronJob. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2149119[*BZ#2149119*])  
```

Comment:
There's a mixture of "cron jobs" and "CronJob". You could change them all to lower case "cron job(s)" (no markup) as a general reference, or for the later ones that mention the API version you could do "v1beta1 `CronJob` objects". See general reference vs object reference in the [API object formatting](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) repo guidelines.

You might also consider marking up `v1beta1`, as I see it often is elsewhere in the repo. E.g., "that support the `v1beta1` API version of `CronJob` objects".

Reply:
@adellape Thank you! Please take a look now.